java/adapter/jdbc/src/test/java/org/apache/arrow/adapter/jdbc/h2/JdbcToArrowVectorIteratorTest.java

```diff
@@ -119,7 +134,10 @@ public void testVectorSchemaRootReuse() throws SQLException, IOException {
       assertNotNull(cur);
 
       // verify the first column, with may contain nulls.
-      assertEquals(expectedColValues[batchCount], cur.getVector(0).toString());
+      assertEquals(expectedIntColValues[batchCount], cur.getVector(0).toString());
```

Comment:
Is there a better way to assert equality than the string representation (perhaps with getObject as done elsewhere)? This is a little brittle

Reply:
Switched to use the `assertIntVectorValues` and `assertListVectorValues`